tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Steps/OrderSteps.cs

```diff
@@ -47,10 +48,10 @@ public async Task GivenOrdersExit(Table table)
             }
         }
 
-        [When(@"a GET request is made for orders")]
-        public async Task WhenAGETRequestIsMadeForOrders()
+        [When(@"a GET request is made for a list of orders with organisationId (.*)")]
+        public async Task WhenAGETRequestIsMadeForOrders(Guid organisationId)
         {
-            await _request.GetAsync(_orderUrl);
+            await _request.GetAsync($"{_orderUrl}/{organisationId}/orders");
```

Comment:
I would be tempted to include the path `/{organisationId}/orders` in the orderUrl as this is specific to the list orders endpoint. All other endpoints for an order will not include the `/organisations` path.

Reply:
👍